src/build_context.rs

```diff
@@ -392,6 +392,12 @@ impl BuildContext {
             fs::copy(&lib_path, &dest_path)?;
             libs_copied.insert(lib_path);
 
+            // fs::copy copies permissions as well, and the original
+            // file may have been read-only
+            let mut perms = fs::metadata(&dest_path)?.permissions();
+            perms.set_readonly(false);
+            fs::set_permissions(&dest_path, perms)?;
```

Comment:
I thought about only setting the permission when needed, but didn't think that it worth the extra if condition to spare a few syscalls, as a stat is needed for each file anyway.